pygmt/base_plotting.py

```diff
@@ -1918,3 +1918,197 @@ def update_pointers(data_pointers):
+                for the wedge, and **-E** to set the color or shade for the
+                uncertainty. Parameters are expected to be in the
+                following columns:
+                    **1,2**: longitude, latitude, of station (-: option
+                    interchanges order) **3**: rotation in radians **4**: rotation
+                    uncertainty in radians
+
+            "**x**\\ *cross_scale*"
+
+                gives Strain crosses. cross_scale sets the size of the cross.
+                Parameters are expected to be in the following columns:
+                    **1,2**: longitude, latitude, of station
+                    (-: option interchanges order) 3: eps1, the most extensional
+                    eigenvalue of strain tensor, with extension taken positive.
+                    **4**: eps2, the most compressional eigenvalue of strain
+                    tensor, with extension taken positive.
+                    **5**: azimuth of eps2 in degrees CW from North.
+
+
+        Other Parameters
+        ----------------
+        vector : bool or str
+            Modify vector parameters. For vector heads, append vector head size
+            [Default is 9p]. See `Vector Attributes <gmt-docs:supplements/geodesy/velo.html#vector-attributes>`_
+            for specifying additional attributes.
+
+        {B}
+
+        {CPT}
+
+        rescale : str
+            can be used to rescale the uncertainties of velocities
+            (-Se and -Sr) and rotations (-Sw).
+            Can be combined with the confidence variable.
+
+        uncertainty_color : str
+            Sets the color or shade used for filling uncertainty wedges
+            (-Sw) or velocity error ellipses (-Se or -Sr). [If -E is not
+            specified, the uncertainty regions will be transparent.]
+            `(more ...) <https://docs.generic-mapping-tools.org/latest/cookbook/features.html#gfill-attrib>`_
+
+        facecolor : str
+            Select color or pattern for filling of symbols or polygons
+            [Default is no fill]. `(more ...) <https://docs.generic-mapping-tools.org/latest/cookbook/features.html#gfill-attrib>`_
+
+        line: str
+            Draw lines. Ellipses and fault planes will have their outlines drawn
+            using current pen (see -W).
+
+        no_clip: str
+            Do NOT skip symbols that fall outside the frame boundary specified
+            by -R. [Default plots symbols inside frame only].
+
+        pen : str
+            Set pen attributes for velocity arrows, ellipse circumference and
+            fault plane edges. [Defaults: width = default, color = black, style = solid].
+
+        {U}
+
+        {V}
+
+        xshift: str
+            ``'[a|c|f|r][xshift]'``
+
+        yshift: str
+            ``'[a|c|f|r][yshift]'``
+            Shift plot origin. `(more ...) <https://docs.generic-mapping-tools.org/latest/gmt.html#xy-full>`_
+
+
```

Comment:
```suggestion
        {U}
        {V}
        {XY}
        {p}
        {t}
```

Common aliases were added in #624 (XY), #627 (p) and #614 (t) so this simplifies things!